x-pack/plugins/security_solution/public/common/components/and_or_badge/index.test.tsx

```diff
@@ -10,8 +10,10 @@ import { ThemeProvider } from 'styled-components';
 import { mount } from 'enzyme';
 
 import { AndOrBadge } from './';
+import { getMockTheme } from '../../mock';
+import { EuiTheme } from '../../../../../../../src/plugins/kibana_react/common';
 
-const mockTheme = { eui: { euiColorLightShade: '#ece' } };
+const mockTheme: EuiTheme = getMockTheme({ eui: { euiColorLightShade: '#ece' } });
```

Comment:
You can remove these explicit types; this is part of the utility of `getMockTheme`.

Reply:
updated